skyline-vscode/src/extension.ts

```diff
@@ -55,73 +42,35 @@ export function activate(context: vscode.ExtensionContext) {
 				let sess_options: SkylineSessionOptions = {
 					context: 		context,
 					projectRoot: 	uri[0].fsPath,
+					// TODO mshin: make the address and port user configurable
 					addr: 			"localhost",
 					port: 			60120,
 					webviewPanel: 	panel
 				};
 
 				let on_close = function() {
-					console.log(sess.backendShouldRestart);
-					if (sess.backendShouldRestart) {
-						console.log("Process closed, restarting.");
+					console.log(sess.resetBackendConnection);
+					console.log("Closing connection to backend");
+					if (sess.resetBackendConnection) {
+						console.log("Reconnecting to backend.");
```

Comment:
are these debug statements? Should we remove them?

Reply:
Just keeping these comments for now. I will remove all calls to console.log once we're done refactoring